oslc-java-samples/pom.xml

```diff
@@ -11,7 +11,7 @@
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
         <wink.version>1.4</wink.version>
-        <lyo.version>2.3.0</lyo.version>
+        <lyo.version>2.4.0</lyo.version>
```

Comment:
the samples have been moved to https://github.com/OSLC/lyo-samples/tree/master/lyo-client-samples

we need to remove them from this repo